src/pages/dev/amplifier/chain-integration/relay-messages.mdx

```diff
@@ -0,0 +1,157 @@
+1. For chains that rely on verifier votes, the Axelar gateway calls [`verify_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L9) on the voting verifier, which prompts verifiers to begin voting.
+1. Once a threshold of successful votes have been cast for the message, an event is emitted.
+1. A relayer hears the event and calls [`route_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L14) on the chain’s Axelar gateway.
+1. The chain’s Axelar gateway passes the messages to the Amplifier router.
+1. The router passes the message to the destination chain’s Axelar gateway.
+1. The chain’s prover calls [`construct_proof`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/contracts/multisig-prover/src/msg.rs#L61), which starts the process of creating a signed batch that can be relayed back to the source chain. This method also passes the now-outbound message from the gateway to the prover.
+1. The prover starts a signing session with the multisig contract.
+1. Verifiers participate in the signing session.
+1. Once enough signatures have been submitted, the relayer gets the fully signed proof from the prover and relays the proof to the destination chain to approve the transactions.
+1. A relayer can now execute any approved transactions on the destination chain.
+
+## Axelar contract deployments
+
+- [`devnet-verifiers.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/9eb7341088cc0a2fd55e76377cd9b2a14f0989e9/axelar-chains-config/info/devnet-verifiers.json)
+- [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json)
+
+## Verify messages
+
+<Callout>
+This is temporarily permissioned to the admin contract established by the network.
+</Callout>
+
+[`verify_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L9) is a command on the gateway that prompts verifiers to begin voting so that a message can be verified and then routed. It takes a vector of messages to be verified.
+
+```bash
+export SOURCE_CHAIN_GATEWAY="axelar1vnfn0e4vnn58ydpm05wqcc9zp8t5ztwd5rw5f895lykqaezmuccqujmwl2"
+export CHAIN_NAME="avalanche"
+export TX_HASH="0x711f8485591d0350dc87f9cb1e807887b91c3fc197f15b95e692a5170146113f"
+export TX_EVENT="0"
+export DESTINATION_CHAIN="ethereum-sepolia"
+export DESTINATION_ADDRESS="0x8f8dedd09E23E22E1555e9D2C25D7c7332291919"
+export SOURCE_ADDRESS="0x0a3b8dc7706c47b6dd87d771df63875b1c5cd867"
+export PAYLOAD_HASH="220f68445e3cec114bff50cd6b251e3deabc7684b10280c2116b20bcc6795a96"
+export RPC="http://devnet-verifiers.axelar.dev:26657"
```

Comment:
Is this a doc for verifiers or integrators? We should make sure we're using the right network & endpoints.